src/NuGet.Core/NuGet.PackageManagement/PublicAPI.Shipped.txt

```diff
@@ -632,9 +632,6 @@ NuGet.ProjectManagement.Preprocessor
 NuGet.ProjectManagement.Preprocessor.Preprocessor() -> void
 ~NuGet.ProjectManagement.Preprocessor.RevertFileAsync(System.Func<System.Threading.Tasks.Task<System.IO.Stream>> streamTaskFactory, string targetPath, System.Collections.Generic.IEnumerable<NuGet.ProjectManagement.InternalZipFileInfo> matchingFiles, NuGet.ProjectManagement.IMSBuildProjectSystem projectSystem, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task
 ~NuGet.ProjectManagement.Preprocessor.TransformFileAsync(System.Func<System.Threading.Tasks.Task<System.IO.Stream>> streamTaskFactory, string targetPath, NuGet.ProjectManagement.IMSBuildProjectSystem projectSystem, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task
-NuGet.ProjectManagement.ProjectBuildProperties
-NuGet.ProjectManagement.ProjectItemProperties
-NuGet.ProjectManagement.ProjectItems
```

Comment:
@nkolev92 (or indeed anyone else), do you think we should add type forwarding for these classes?

NuGet.PackageManagement is just such an unlikely package for anyone interacting with NuGet to use, so it seems incredibly unlikely that anyone actually depends on these APIs.   Plus, since they're `const string`, I'm not even use if there's a runtime dependency on the type, or if the compiler copies the const string into the assembly being compiled.

Reply:
I'd say disclaimer applies, mark the issue, make sure it's in the release notes and it's all good.